docs/resources/dns_filtering_profile.md

```diff
@@ -0,0 +1,151 @@
+---
```

Comment:
I think we need to add some text there on how this is a business or enterprise feature and it needs to be enabled before you can use it